### PR TITLE
Clarify actuator security documentation

### PR DESCRIPTION
**Pull request by cmabdullah**

### CODE REVIEW

**Review by wilkinsona — CHANGES_REQUESTED** [review]
Thanks for the pull request, @cmabdullah. I've left a couple of comments for your consideration.

File: spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/endpoints.adoc [hunk 1]
```diff
@@ -316,8 +316,18 @@ TIP: If you want to implement your own strategy for when endpoints are exposed,
...
+
+The following configuration expose `/info` and `/metrics` endpoint from the example.com domain:
+[source,yaml,indent=0,subs="verbatim",configprops,configblocks]
+----
+	management:
+	  endpoints:
+	    web:
+	      exposure:
+	        include: ["info", "metrics"]
+----
```
[the remark this answers is not in the document]
Comment (cmabdullah) [reply spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/endpoints.adoc]: Example removed accordingly.

File: spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/endpoints.adoc [hunk 2]
```diff
@@ -316,8 +316,18 @@ TIP: If you want to implement your own strategy for when endpoints are exposed,
 
 [[actuator.endpoints.security]]
 === Security
-For security purposes, all actuators other than `/health` are disabled by default.
-You can use the configprop:management.endpoints.web.exposure.include[] property to enable the actuators.
+For security purposes, all actuator's endpoints that are exposed over HTTP are secret by default except `/health` endpoint.
```
[the remark this answers is not in the document]
Comment (cmabdullah) [reply spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/endpoints.adoc]: Updated according to your suggestions, thanks for your concern.

**Comment by wilkinsona** [comment]
@cmabdullah Thank you for making your first contribution to Spring Boot.

**Pull request closed without merging**